openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -460,11 +461,14 @@ def get_all_courses(cls, org=None, filter_=None):
         # created. For tests using CourseFactory, use emit_signals=True.
         course_overviews = CourseOverview.objects.all()
 
-        if org:
+        if orgs:
             # In rare cases, courses belonging to the same org may be accidentally assigned
             # an org code with a different casing (e.g., Harvardx as opposed to HarvardX).
-            # Case-insensitive exact matching allows us to deal with this kind of dirty data.
-            course_overviews = course_overviews.filter(org__iexact=org)
+            # Case-insensitive matching allows us to deal with this kind of dirty data.
+            # We need to escape the org names provided in case they contain regex special
+            # chars for some reason.
```

Comment:
Was this actually a problem?  Do we have a test value to cover this scenario?  The names I see in the included tests don't appear to contain any special regex characters (but I might have missed it)